src/site/xdoc/checks/coding/variabledeclarationusagedistance.xml

```diff
@@ -275,8 +278,10 @@ public class Example4 {
 public class Example5 {
 
   public void foo1() {
-    int num;        // violation, distance = 4
-    final double PI;   // violation, distance = 5
+    // violation below, 'variable 'num' declaration and its first usage is 4.'
```

Comment:
```suggestion
    // violation below, declaration and first usage of 'variable 'num' is 4.'
```
keeping things together like `story and telling` and `vars and number` improves the story cohesion.